src/pages/App.tsx

```diff
@@ -1,5 +1,5 @@
-import "../styles/global.scss"
-import "./NotifyStyle.scss"
+// import "../styles/global.scss"
+// import "./NotifyStyle.scss"
```

Comment:
Lets use this PR to simply set up all the boilerplate (creating the themes, removing chakra, creating a new theme provider, syncing with system-mode), and then use a separate PR to start writing and using new components.

Reply:
Because this global style makes web page don't render after adding MUI.
That is the reason why I commented out these import.